src/Files.App/ViewModels/UserControls/ToolbarViewModel.cs

```diff
@@ -17,6 +17,8 @@ namespace Files.App.ViewModels.UserControls
 {
 	public class ToolbarViewModel : ObservableObject, IAddressToolbar, IDisposable
 	{
+		private const int MAX_SUGGESTIONS = 7;
```

Comment:
Can we increase this to 10 to match the number of suggestions we show for the search box?

Reply:
👍